opm/simulators/linalg/bda/BdaBridge.cpp

```diff
@@ -193,6 +193,8 @@ void BdaBridge<BridgeMatrix, BridgeVector, block_size>::copySparsityPatternFromI
 
 template <class BridgeMatrix, class BridgeVector, int block_size>
 void BdaBridge<BridgeMatrix, BridgeVector, block_size>::solve_system([[maybe_unused]] BridgeMatrix* mat,
+                                                                     [[maybe_unused]] BridgeMatrix* blockMat,
+                                                                     [[maybe_unused]] int numJacobiBlocks,
```

Comment:
Use `size_t`? Why `[[maybe_unused]]`? Seems like both are used in the code (BTW: might hold for the remaining parameters, too).

Reply:
Do you mean the param `numJacobiBlocks`?
It is read by `EWOMS_GET_PARAM(TypeTag, int, NumJacobiBlocks)`, that's probably why it's an `int`.
`[[maybe_unused]]` is removed.